src/routes/docs/advanced/self-hosting/environment-variables/+page.markdoc

```diff
@@ -29,7 +29,11 @@ docker compose exec appwrite vars
 | `_APP_OPENSSL_KEY_V1` | This is your server private secret key that is used to encrypt all sensitive data on your server. Appwrite server encrypts all secret data on your server like webhooks, HTTP passwords, user sessions, and storage files. The var is not set by default, if you wish to take advantage of Appwrite encryption capabilities you should change it and make sure to **keep it a secret and have a backup for it**. |
 | `_APP_DOMAIN` | Your Appwrite domain address. When setting a public suffix domain, Appwrite will attempt to issue a valid SSL certificate automatically. When used with a dev domain, Appwrite will assign a self-signed SSL certificate. The default value is 'localhost'. |
 | `_APP_DOMAIN_FUNCTIONS` | A domain to use for function preview URLs. Setting to empty turns off function preview URLs. |
+| `_APP_DOMAIN_SITES` | **version >= 1.7.0** The domain to use for site URLs. Setting to empty turns off site URLs. |
 | `_APP_DOMAIN_TARGET` | A DNS A record hostname to serve as a CNAME target for your Appwrite custom domains. You can use the same value as used for the Appwrite '_APP_DOMAIN' variable. The default value is 'localhost'. |
+| `_APP_DOMAIN_TARGET_CNAME` | **version >= 1.7.0** A DNS CNAME record hostname to serve as a CNAME target for your Appwrite custom domains. You can use the same value as used for the Appwrite '_APP_DOMAIN_TARGET' variable. The default value is 'localhost'. |
```

Comment:
_APP_DOMAIN_TARGET above can be removed now, we now use _APP_DOMAIN_TARGET_CNAME as exact replacement

Reply:
Added a deprecation note